### PR TITLE
Added strimzi installation.

### PR DESCRIPTION
**Pull request by oleksii-kondratiuk**

### CODE REVIEW

File: tools-istio/strimzi.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nordmart-dev-strimzi
```
Comment (ahmedwaleedmalik) [inline tools-istio/strimzi.yaml]: `nordmart-dev-strimzi-kafka-operator`

File: tools-istio/strimzi.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  namespace: nordmart-dev-tools
+spec:
+  releaseName: nordmart-dev-strimzi
```
Comment (ahmedwaleedmalik) [inline tools-istio/strimzi.yaml]: `nordmart-dev-strimzi-kafka-operator`

**Review by ahmedwaleedmalik — CHANGES_REQUESTED** [review]
I'd suggest if we could go with `kafka-operator` instead of `strimzi` everywhere. If you think that's appropriate please make the change

File: tools-istio/strimzi.yaml (hunk 1 above)
Comment (oleksii-kondratiuk) [reply tools-istio/strimzi.yaml]: Agree on the naming, changed.

File: tools-istio/strimzi.yaml (hunk 2 above)
Comment (oleksii-kondratiuk) [reply tools-istio/strimzi.yaml]: Agree on the naming, changed.

File: tools-istio/strimzi.yaml (hunk 1 above)
Comment (ahmedwaleedmalik) [reply tools-istio/strimzi.yaml]: Thanks

**Review by ahmedwaleedmalik — APPROVED** [review]

**Pull request merged**